### ISSUE TITLE
Add light theme

### ISSUE THREAD

**Issue by vladshcherbin** [Undre4m/igniculus#2]
It would be cool to have a light theme out of the box, there are a lot of white terminals. Currently it looks like this:

![screen shot 2018-05-30 at 12 55 52](https://user-images.githubusercontent.com/6711845/40713581-fb406080-6408-11e8-82e5-1d8ca9485e5c.png)

**Comment by Undre4m** [comment]
It would be nice to have a couple of pre-built themes.
I'm thinking something like `igniculus('light')`,
`'light'` being the name of one of the available themes ofc...

I'll add it as a planned feature for next release. Default theme would remain the same so its non-breaking.

**Comment by vladshcherbin** [comment]
@Undre4m maybe it's better to make it as an option, to be like the other options. Style related options to define in a nested group, smth like:

```js
igniculus({
  theme: 'light',
  styles: {
    comments: { mode: 'dim', fg: 'white' },
    constants: { mode: 'dim', fg: 'red' },
    ...
  },
  prefix: ...,
  postfix: ...,
  output: ...
})
```

Yes, it's a breaking one, but a good one for a future run since more options may be needed later and now options are already in mess (styles, prefix, postfix, output together).

What do you think?

**Comment by Undre4m** [comment]
@vladshcherbin yes, separation of style-related and option-specific configurations is something i've been wanting to do but haven't had the time. It's an issue for version 2.0, among others. Probably something like:
```javascript
igniculus({
  prefix: {
    style: { fg: ... },
    options: {  text: ... }
  }
})
```

I do like the idea of a **theme** option, i think is the way to go; defining an optional pre-built theme to use, and being able to add/override specific options by the usual manner.
